docs/source/features/data-sources.md

```diff
@@ -228,7 +228,7 @@ class PersonalizationAPI extends RESTDataSource {
 
 ## Using Memcached/Redis as a cache storage backend
 
-By default, resource caching will use an in memory LRU cache. When running multiple server instances, you'll want to use a shared cache backend instead. That's why Apollo Server also includes support for using [Memcached](../../../packages/apollo-server-cache-memcached) or [Redis](../../../packages/apollo-server-cache-redis) as your backing store. You can specify which one to use by creating an instance and passing it into the Apollo Server constructor:
+By default, resource caching will use an in memory LRU cache. When running multiple server instances, you'll want to use a shared cache backend instead. That's why Apollo Server also includes support for using [Memcached](https://memcached.org/) or [Redis](https://redis.io/) as cache stores via the [`apollo-server-cache-memcached`](https://www.npmjs.com/package/apollo-server-cache-memcached) and [`apollo-server-cache-redis`](https://www.npmjs.com/package/apollo-server-cache-redis) packages. You can specify which one to use by creating an instance and passing it into the `ApolloServer` constructor:
```

Comment:
> in memory LRU cache

=> `in-memory LRU cache`

Reply:
Good call.  I'll push a separate commit with that fix after this merges since that change isn't introduced by or related to this PR.